qa/pull-tester/rpc-tests.sh

```diff
@@ -8,6 +8,11 @@ CURDIR=$(cd $(dirname "$0"); pwd)
 export BITCOINCLI=${BUILDDIR}/qa/pull-tester/run-bitcoin-cli
 export BITCOIND=${REAL_BITCOIND}
 
+if [ "x${EXEEXT}" = "x.exe" ]; then
+  echo "Win tests currently disabled"
+  exit 0
+fi
```

Comment:
Nope, they are still disabled by default upstream, and https://github.com/bitcoin/bitcoin/pull/5677#issuecomment-136646964 is referenced as the reason for disabling. In upstream's migration of this script from Bash to Python (bitcoin/bitcoin#6616), an override flag was added, so we will gain that as well once I pull that in.